### PR TITLE
Add isGroupCulled property to ModelOverlays

### PR DESCRIPTION
**Pull request by SamGondelman**
And enable it on the tablet model so that the X button isn't hidden by LOD: https://highfidelity.fogbugz.com/f/cases/14535/X-button-on-tablet-is-hidden-by-low-LOD

The highlight is still hidden by LOD because it's a separate overlay.  We've decided not to fix this at this time.

Test plan:
- Using Developer -> Render -> LOD Tools, manually adjust your LOD to be lower.  In master, the "x" button on the tablet would disappear way before the rest of the tablet, but in this PR the "x" button shouldn't disappear until the entire tablet is hidden.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10339/HighFidelity-Beta-PR13294-c8669f9.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10339/HighFidelity-Beta-PR13294-c8669f9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10339/HighFidelity-Beta-Interface-PR13294-c8669f9.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10340/HighFidelity-Beta-PR13294-bde3382.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10340/HighFidelity-Beta-PR13294-bde3382.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10340/HighFidelity-Beta-Interface-PR13294-bde3382.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10340/HighFidelity-Beta-PR13294-bde3382.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10340/HighFidelity-Beta-Interface-PR13294-bde3382.exe

File: interface/src/ui/overlays/ModelOverlay.cpp [hunk 1]
```diff
@@ -158,6 +162,11 @@ void ModelOverlay::setDrawHUDLayer(bool drawHUDLayer) {
     _drawInHUDDirty = true;
 }
 
+void ModelOverlay::setGroupCulled(bool groupCulled) {
+    _isGroupCulled = groupCulled;
+    _groupCulledDirty = true;
+}
```
Comment (AndrewMeadows) [inline interface/src/ui/overlays/ModelOverlay.cpp]: Maybe this should be:
```
void ModelOverlay::setGroupCulled(bool groupCulled) {
    if (_isGroupCulled != groupCulled) {
        _isGroupCulled = groupCulled;
        _groupCulledDirty = true;
    }
}
```
Or maybe it doesn't matter?  What is the overhead of calling `Model::setGroupCulled()` unnecessarily in the `update()` method?

File: interface/src/ui/overlays/ModelOverlay.cpp [hunk 2]
```diff
@@ -347,6 +361,8 @@ vectorType ModelOverlay::mapJoints(mapFunction<itemType> function) const {
  *     {@link Overlays.findRayIntersection|findRayIntersection} ignores the overlay.
  * @property {boolean} drawInFront=false - If <code>true</code>, the overlay is rendered in front of other overlays that don't
  *     have <code>drawInFront</code> set to <code>true</code>, and in front of entities.
+ * @property {boolean} isGroupCulled=false - If <code>true</code>, the mesh parts of the model are LOD culled as a group.
+ *     If <code>false</code>, separate mesh parts will be LOD culled individually.
```
Comment (AndrewMeadows) [inline interface/src/ui/overlays/ModelOverlay.cpp]: So... setting `isGroupCulled` property is a something we expect normal users to actually do?  Or we are adding this as hackery to get us out of a corner we painted ourselves into?

File: interface/src/ui/overlays/ModelOverlay.cpp (hunk 1 above)
Comment (SamGondelman) [reply interface/src/ui/overlays/ModelOverlay.cpp]: It's not particularly expensive but I've gone ahead and added that check here and in the similar methods above just for clarity.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10357/HighFidelity-Beta-PR13294-968758a.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10357/HighFidelity-Beta-PR13294-968758a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10357/HighFidelity-Beta-Interface-PR13294-968758a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10357/HighFidelity-Beta-PR13294-968758a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10357/HighFidelity-Beta-Interface-PR13294-968758a.exe

**Pull request merged**